lighthouse-extension/gulpfile.js

```diff
@@ -119,7 +119,6 @@ gulp.task('browserify-lighthouse', () => {
       // Do the additional transform to convert references of devtools-timeline-model
       // to the modified version internal to Lighthouse.
       bundle.transform('./dtm-transform.js', {global: true})
-      .ignore('../lighthouse-core/lib/asset-saver.js') // relative from gulpfile location
```

Comment:
I thought this might pull in a scary amount of stuff into the bundle, but bundleSize doesn't seem to think so, so yay! :)